app/src/main/java/org/p2p/wallet/auth/repository/FileRepository.kt

```diff
@@ -47,6 +41,7 @@ class FileRepository(private val context: Context) {
 
     private fun ensurePdfFolderExists() = pdfFolder.mkdirs()
 
+    // TODO remove if not needed!
```

Comment:
todo?

Reply:
yes i think we need to remove this method as not needed!